app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -160,7 +138,12 @@ export class MergeConflictsDialog extends React.Component<
       },
     })
     this.props.dispatcher.recordMergeConflictsDialogDismissal()
-    if (getConflictedFiles(this.props.workingDirectory).length > 0) {
+    if (
+      getConflictedFiles(
```

Comment:
Extracting this bit to a named constant would help make the intent of this call clearer to me.